### DISCUSSION TITLE
Not sure how to use my SMB share in retronas?

### DISCUSSION THREAD

**Discussion by hawkeye73** [retronas/discussions#24] [Q&A]
Hi,

I installed retronas but I'm not sure how I can use my SMB shares from my 'main' NAS to use in RetroNAS. I would love to use my already existing SMB share with all files on RAID6 in RetroNAS. How can I do this without altering my SMB share?

Cheers

**Comment by hawkeye73** [comment]
anyone?

**Reply by sairuk** [reply]
This is a solution very specific to your own setup, without detail on your setup we cant really even begin to suggest a solution

You would (perhaps) in this case mount your remote storage in place onto RetroNAS with a cifs mount in the RetroNAS fstab (if you want to use samba) then let RetroNAS do its thing with the storage. 

The RetroNAS directory structure will still be a requirement however and while is should not interfere with your existing nas structure we cannot be certain of that.

**Reply by hawkeye73** [reply]
thanks, I will wait a bit I can't risk changing my own structure atm.

**Comment by tanuki2k** [comment]
I install retronas as a VM on my Synology NAS then used a combo of NFS and symlink to avoid having to move ROMS from my original file location/shares to retronas's folder structure. So far it works well. Though all I'm only really using RETRONAS for GameCube, ps2, and file server for my old macs and windows xp computer.

**Reply by drinksomewhiskey** [reply]
Were you able to mount a shared folder on the Synology NAS via NFS share and use it with retronas without any issues? I am able to mount the NFS share via cockpit, but any service I try to install via retronas fails due to permissions issues. How did you get this to work? On the Synology side, I allowed connected users to be connected as root users and on the cockpit side I am not selecting the read only option (thus it should be able to read/write). Any idea on what I'm missing here? Any info would help!

**Comment by danmons** [comment]
I've got on my to-do list a task to document how to re-export an existing, underlying SMB share with RetroNAS SMB.

As @sairuk said, it will be VERY specific to your own NAS.  I'll try to document it as generically as possible, but there will be elements about the underlying NAS that will be vender-specific, but I'll try my best to make it obvious as to what's expected.
